components/homme/src/share/control_mod.F90

```diff
@@ -617,4 +632,73 @@ subroutine test_timestep_make_parameters_consistent(par, nerr)
          write(iulog,'(a,i2)') 'test_timestep_make_parameters_consistent nerr', nerr
   end subroutine test_timestep_make_parameters_consistent
 
+
+subroutine set_planar_defaults()
+
+!if true, most likely lx,ly,sx,sy weren't set in ctl_nl
+    if (      abs(lx).le.tol_zero .and. abs(ly).le.tol_zero &
+        .and. abs(sx).le.tol_zero .and. abs(sy).le.tol_zero )then
+
+    if (test_case == "planar_dbl_vrtx") then
+      Lx = 5000.0D0 * 1000.0D0
+      Ly = 5000.0D0 * 1000.0D0
+      Sx = 0.0D0
+      Sy = 0.0D0
+    else if (test_case == "planar_hydro_gravity_wave") then
+       Lx = 6000.0D0 * 1000.0D0
+       Ly = 6000.0D0 * 1000.0D0
+       Sx = -3000.0D0 * 1000.0D0
+       Sy = -3000.0D0 * 1000.0D0
+    else if (test_case == "planar_nonhydro_gravity_wave") then
+       Lx = 300.0D0 * 1000.0D0
+       Ly = 300.0D0 * 1000.0D0
+       Sx = -150.0D0 * 1000.0D0
+       Sy = -150.0D0 * 1000.0D0
+    else if (test_case == "planar_hydro_mtn_wave") then
+       Lx = 240.0D0 * 1000.0D0
+       Ly = 240.0D0 * 1000.0D0
+       Sx = 0.0D0 * 1000.0D0
+       Sy = 0.0D0 * 1000.0D0
+    else if (test_case == "planar_nonhydro_mtn_wave") then
+       Lx = 144.0D0 * 1000.0D0
+       Ly = 144.0D0 * 1000.0D0
+       Sx = 0.0D0 * 1000.0D0
+       Sy = 0.0D0 * 1000.0D0
+    else if (test_case == "planar_schar_mtn_wave") then
+       Lx = 100.0D0 * 1000.0D0
+       Ly = 100.0D0 * 1000.0D0
+       Sx = 0.0D0 * 1000.0D0
+       Sy = 0.0D0 * 1000.0D0
+    else if (test_case == "planar_density_current" .OR. test_case == "planar_moist_density_current") then
+       Lx = 51.2D0 * 1000.0D0
+       Ly = 51.2D0 * 1000.0D0
+       Sx = -25.6D0 * 1000.0D0
+       Sy = -25.6D0 * 1000.0D0
+    else if (test_case == "planar_rising_bubble" .OR. test_case == "planar_moist_rising_bubble") then
+       Lx = 1.0D0 * 10000.0D0
+       Ly = 1.0D0 * 10000.0D0
+       Sx = -5000.0D0
+       Sy = -5000.0D0
+! THESE ARE WRONG AND NEED TO BE FIXED WHEN THESE CASES ARE ACTUALLY IMPLEMENTED....
```

Comment:
Same here- maybe adjust these defaults to match one of the existing bubble cases from the CRM?

Reply:
will do, thanks